src/store/singing.ts

```diff
@@ -1105,32 +1053,35 @@ export const singingStore = createPartialStore<SingingStoreTypes>({
           // Score & AudioQuery -> StartTime
 
           if (!phrase.query) {
-            window.electron.logInfo(`Generating query...`);
+            const engineId = phrase.singer.engineId;
+            const frameRate = state.engineManifests[engineId].frameRate;
+            const restDurationSeconds = 1;
```

Comment:
前後の休符の長さはひとまず1秒に設定しています。

Reply:
あ、必要なのか十分なのかわからないのでメモ書きだけ･･･！
```suggestion
            const restDurationSeconds = 1; // 仮置き
```